Upgrade autoloader and other

bambutcha opened the pull request:
Move Counter.php to Handler folder
Edit FileCsv.php autoloader.php index.php

[#1.2]

bambutcha committed:
Upgrade autoloader and other
Move Counter.php to Handler folder
Edit FileCsv.php autoloader.php index.php

[#1.2]

abutenko3 commented on autoloader.php:
```diff
@@ -1,13 +1,18 @@
 
     foreach ($dirFiles as $file) {
-        require __DIR__ . DIRECTORY_SEPARATOR .implode(DIRECTORY_SEPARATOR, [$dirName, $file]);
+        if ($file !== '.' && $file !== '..') {
```
Го использовать [guard clause](https://deviq.com/design-patterns/guard-clause)

abutenko3 commented on App/FileCsv.php:
```diff
@@ -25,9 +26,16 @@ public function writeData(array $entityBody, string $fileName, array $headers):
+
+    private function getFilePath(string $fileName): string
+    {
+        return implode('/', [self::DEFAULT_OUTPUT_NAME, $fileName . '.' . self::DEFAULT_FILE_FORMAT]);
```
юзаем для путей `DIRECTORY_SEPARATOR`

bambutcha committed:
Include composer's autoloader and fix code
Edit FileCsv.php autoloader.php composer.json index.php
Edit vendor/composer/autoload_psr4.php autoload_static.php

[#1.2]

bambutcha committed:
Update composer's autoloader code
Edit composer.json
update autoload files

[#1.2]

bambutcha committed:
Delete unnecessary files

[#1.2]

bambutcha committed:
Delete unnecessary files from repo

[#1.2]

bambutcha committed:
Add composer.json composer.lock

[#1.2]

bambutcha committed:
Add .gitignore

[#1.2]
